### DISCUSSION TITLE
Metody do definiowania ofert powiązanych [cross-sell]

### DISCUSSION THREAD

**Discussion by kfijalko** [allegro/allegro-api#4397] [Suggestion - verification]
Klientowi wyświetlają się "oferty powiązane" jak doda do koszyka lub jak jest w koszyku, lub na stronie oferty "inne/pozostałe oferty sprzedającego.
Czy planujecie udostępnienie możliwości definiowania "ofert powiązanych".

**Comment by kfijalko** [comment]
Kiedyś można to było robić w "nowym menadżerze sprzedaży".

**Comment by PrzemyslawLukanowski** [comment]
Na tę chwilę nie mamy tego w planach. Poinformujemy, jeśli coś się zmieni w tym temacie.
